CHANGELOG.next.asciidoc

```diff
@@ -125,6 +125,24 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update documentation for system.process.memory fields to include clarification on Windows os's. {pull}17268[17268]
 - When using the `decode_json_fields` processor, decoded fields are now deep-merged into existing event. {pull}17958[17958]
 - Add keystore support for autodiscover static configurations. {pull]16306[16306]
+- Add backoff configuration options for the Kafka output. {issue}16777[16777] {pull}17808[17808]
```

Comment:
Looks like this has some extra lines in it.

Reply:
Is it possible for this to happen during a run of the cherry-pick script? The initial PR does not seem to have it: https://github.com/elastic/beats/commit/3f025e1df0fe45a2bcdc1e5ea44ed3c24a9f899a